### DISCUSSION TITLE
Paginator and masto.v1.followedTags.list example

### DISCUSSION THREAD

**Discussion by GerritKuilder** [neet/masto.js#1242] [Q&A]
Hi All,

I have been spending a long day behind the computer and the last thing I wanted to test is getting all the followed tags for an account.

I can get the first 100 (that's by default, no surprise there) but I am a bit stumped how I can get all the tags the account follows.

I checked various examples mentioning paging and I looked at the code too but have not made any progress.

I tried  const tags2 =await tags.next() as mentioned in another discussion but no luck there.(not a function)

The first 100 I get with const tags = await masto.v1.followedTags.list(); 

But I am stumped after that.

Kind Regards,

Gerrit Kuilder

**Comment by neet** [comment]
@GerritKuilder You may need to write as follows, without `await` at the beginning:

```ts
const tags = masto.v1.followedTags.list()

const tags1 = await tags.next()
const tags2 = await tags.next();
```

I hope this helps.
